lightning/restore/tidb_test.go

```diff
@@ -107,3 +147,188 @@ func (s *tidbSuite) TestCreateTableIfNotExistsStmt(c *C) {
 		"SET NAMES 'binary';SET @@SESSION.`FOREIGN_KEY_CHECKS`=0;CREATE TABLE IF NOT EXISTS `m` (`z` DOUBLE) ENGINE = InnoDB AUTO_INCREMENT = 8343230 DEFAULT CHARACTER SET = UTF8;",
 	)
 }
+
+func (s *tidbSuite) TestInitSchema(c *C) {
+	ctx := context.Background()
+
+	s.mockDB.
+		ExpectExec("CREATE DATABASE IF NOT EXISTS `db`").
+		WillReturnResult(sqlmock.NewResult(1, 1))
+	s.mockDB.
+		ExpectExec("USE `db`").
+		WillReturnResult(sqlmock.NewResult(0, 0))
+	s.mockDB.
+		ExpectExec("\\QCREATE TABLE IF NOT EXISTS `t1` (`a` INT PRIMARY KEY,`b` VARCHAR(200));\\E").
```

Comment:
can you give reference about usage of `\\Q` and `\\E` for me to learn it

Reply:
@july2993 https://godoc.org/regexp/syntax Under "Escape sequences"

```
\Q...\E        literal text ... even if ... has punctuation
```

Basically `"\\Qstuff\\E"` is equivalent to `regexp.QuoteMeta("stuff")`.